eval.c

```diff
@@ -1079,11 +1079,21 @@ rb_mod_prepend(int argc, VALUE *argv, VALUE module)
 {
     int i;
     ID id_prepend_features, id_prepended;
+    VALUE new_mod;
 
     CONST_ID(id_prepend_features, "prepend_features");
     CONST_ID(id_prepended, "prepended");
 
-    rb_check_arity(argc, 1, UNLIMITED_ARGUMENTS);
+    if(rb_block_given_p()) {
+      new_mod = rb_module_new();
```

Comment:
I attempted to more closely match the style in https://github.com/ruby/ruby/pull/1639/commits/0439e5c39f70e4be882464028e4ac8f60c6de16a. There were already two indentation styles so I tried to follow the one that seemed more common.